app/scripts/views/components/shareVirtualStudy/shareVirtualStudy.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * Share virtual cohort component
+ * Created by Jing Su on 9/25/2017
+ */
+
+'use strict';
+(function(Vue, $, vcSession) {
+  Vue.component('shareVirtualStudy', {
+    template:
+    '<div v-if="showShareButton" class="share-virtual-study">' +
+    '<div class="share-cohort-btn">' +
+    '<i class="fa fa-share-alt" alt="Share Virtual Study"></i></div></div>',
+    props: [
+      'selectedPatientsNum', 'selectedSamplesNum',
+      'stats', 'updateStats', 'showShareButton'
+    ],
+    data: function() {
+      return {
+        savedVC: null
+      };
+    }, methods: {
+      saveCohort: function() {
+        var _self = this;
+        _self.updateStats = true;
+        _self.$nextTick(function() {
+          _self.addNewVC = true;
+        });
+      }
+    },
+    ready: function() {
+      var self_ = this;
+      var previousSelectedCases = {};
+      if (this.showShareButton) {
```

Comment:
Should we give a default value to `showShareButton`? Probably we should do the same in showSaveButton

Reply:
I think the default value of showShareButton is true, which is set in vueCore.js right?